x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_generic_errors_list.tsx

```diff
@@ -23,11 +23,10 @@ export const EndpointGenericErrorsList = memo<PackageGenericErrorsListProps>(
     const globalEndpointErrors = useMemo(() => {
       const errors: PackageActionFormatter[] = [];
       packageErrors.forEach((unit) => {
-        if (unit.payload && unit.payload.error) {
+        if (unit.status === 'failed') {
```

Comment:
Will this be always `failed` for errors even when the error is from an es connection or others?

Reply:
yeah, this is an overall status. failure for any reason, whether defined with error code or not, will result in `failed`.